bigbluebutton-html5/imports/startup/client/logger.js

```diff
@@ -0,0 +1,152 @@
+import Auth from '/imports/ui/services/auth';
+import { Meteor } from 'meteor/meteor';
+import { createLogger, stdSerializers } from 'browser-bunyan';
+import { ConsoleFormattedStream } from '@browser-bunyan/console-formatted-stream';
+import { nameFromLevel } from '@browser-bunyan/levels';
+
+// The logger accepts "console","server", and "url" as targets
+// Multiple targets can be set as an array in the settings under public.log.target
+// Set the desired log levels to be sent under public.log.level
+// The accepted levels are "debug", "info", "warn", "error"
+// If sending to a url, provide the end-point on public.log.url
+// Call the logger by doing a function call with the level name, I.e, logger.warn('Hi on warn')
+
+const LOG_CONFIG = Meteor.settings.public.log || {};
+const loggerStreams = []; // Stores the targets streams
+const userAgent = typeof window !== 'undefined' ? window.navigator.userAgent : 'no-window';
+const isBot = /bot|crawler|spider|crawling/i.test(userAgent);
```

Comment:
is there a case where a crawler join our client? if thats the case we need to avoid id via robotstxt